R/bootstrap.R

```diff
@@ -42,16 +42,60 @@ bootstrapPage <- function(..., title = NULL, responsive = NULL, theme = NULL) {
       # remainder of tags passed to the function
       list(...)
     ),
-    if (useBsTheme()) bootstraplib::bootstrap() else bootstrapLib()
+    bootstrapLib_()
   )
 }
 
+#' Bootstrap libraries
+#'
+#' This function is deprecated in favor of [bootstraplib::bootstrap()], which
+#' provides a more fully featured approach to accessing and theming Bootstrap
+#' CSS. Note that most users won't need to call this function directly since
+#' [bootstrapPage()] and it's special cases ( e.g., [fluidPage()],
+#' [basicPage()], [navbarPage()], etc) implicitly depend on Bootstrap.
+#'
+#' @param theme deprecated (it never worked).
+#' @export
+#' @keywords internal
+bootstrapLib <- function(theme = NULL) {
+  if (!is.null(theme)) {
+    stop("shiny::bootstrapLib()'s theme argument never worked as intended, don't use it.")
```

Comment:
I think this should be a message -- if people are currently using it, no need to make their app stop working.

Reply:
Ok, yea.